doc/requirements.txt

```diff
@@ -0,0 +1,4 @@
+jupyter_client
```

Comment:
It is quite common practice though to put a `requirements.txt` file in a folder that specifies the dependencies required to get your project/subfolder up and running. This simplifies the setup procedure slightly. This should not be used as a replacement for `setup.py` though since they serve a different purpose. 

https://packaging.python.org/discussions/install-requires-vs-requirements/#requirements-files

Yes this is supported by readthedocs.

<img width="617" alt="Schermafbeelding 2019-08-28 om 07 54 42" src="https://user-images.githubusercontent.com/326308/63829218-333e9800-c969-11e9-9255-f8a2baa64536.png">